string name

bool alive 
uint32 restart_count

# percent of cpu usage
float32 cpu

# cpu usage times
float32 user 
float32 system 
float32 children_user 
float32 children_system
float32 iowait

# percent of memory based on rss
float32 memory

# bytes of memory used
uint64 rss
uint64 pss
uint64 uss 
uint64 swap
